src/templates/layouts/main.handlebars

```diff
@@ -0,0 +1,21 @@
+<html>
+    <head>
+        <title>Developers Preference Testing Tool</title>
```

Comment:
Shall this title be fetched from the message bundle too?

Reply:
done